db/migrate/20170829044249_create_events.rb

```diff
@@ -0,0 +1,10 @@
+class CreateEvents < ActiveRecord::Migration[5.1]
+  def change
+    create_table :events do |t|
+      t.string :name, index: true, unique: true, null: false
+      t.datetime :time, index: true, null: false
+      t.integer :duration, index: true, null: false
```

Comment:
I was thinking `:duration` should be a float to represent hours. That way, events could be 30 minutes, an hour and a half, etc.

Reply:
I think the issue with using a float is then you can put in durations like 1.14 hours.

Reply:
Fair enough, so as is, each unit would be 30 minutes?

Reply:
That's a good point, I assumed each digit was a minute.  So duration of 30 for 30 minutes.  We should make a decision and either write a comment or change the column name to express what the unit is.

Reply:
Yeah, I like that idea. Just rename the column `duration_minutes`

Reply:
I vote for changing it to `minutes`